Source/WTF/wtf/text/LChar.h

```diff
@@ -26,7 +26,95 @@
 
 #pragma once
 
-// A type to hold a single Latin-1 character.
-// This type complements the char16_t type that we get from C++.
-// To parallel that type, we put this one in the global namespace.
-typedef unsigned char LChar;
+#include <wtf/StdLibExtras.h>
+
+namespace WTF {
+
+// Single Latin-1 character. Complements char8_t for UTF-8, char16_t for UTF-16, char32_t for UTF-32.
+struct LChar {
+    uint8_t value;
+
+    LChar() = default;
+    constexpr LChar(std::integral auto value) : value { static_cast<uint8_t>(value) } { }
+    constexpr operator std::integral auto() const { return value; }
+    constexpr bool operator !() const { return !value; }
+    constexpr bool operator ==(const LChar&) const = default;
+    constexpr auto operator <=>(const LChar&) const = default;
```

Comment:
For consistency with operators in the rest of the project these probably shouldn't have a space between the `operator` and the symbol

Reply:
I think I ran into a lot of style script errors until I added the spaces. I can try again.